frontend/src/lib/components/PropertyFilters/Filter/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { SelectGradientOverflowProps } from 'lib/components/SelectGradientOverflow'
+import { propertyFilterLogic } from '../propertyFilterLogic'
+import { TabbedPropertyFilter } from './TabbedPropertyFilter'
+import { UnifiedPropertyFilter } from './UnifiedPropertyFilter'
+
+interface PropertyFilterProps {
```

Comment:
Seems like we're repeating this interface in multiple places, why don't define it here, and reuse it in the downstream components? Maybe just extend it as appropriate